game.ini

```diff
@@ -16,110 +16,113 @@ health_potion_1=0,100,76,10,255,255
-mana_top=915
-mana_left=1442
-mana_width=10
-mana_height=151
-potion1_x=1113
-potion1_y=1042
-potion_width=45
-potion_height=45
-potion_next=62
-merc_health_top=21
-merc_health_left=23
-merc_health_width=60
-merc_health_height=5
+health_top=610
+health_left=309
+health_width=7
+health_height=101
+mana_top=610
+mana_left=961
+mana_width=7
+mana_height=101
+potion1_x=742
+potion1_y=695
+potion_width=30
+potion_height=30
+potion_next=41
+merc_health_top=14
+merc_health_left=15
+merc_health_width=40
+merc_health_height=3
 ; skills
-skill_y=1040
-skill_right_x=1030
-skill_left_x=895
-skill_width=62
-skill_height=62
+skill_y=693
+skill_right_x=687
+skill_left_x=597
+skill_width=41
+skill_height=41
 ; death screen
-death_roi_top=290
-death_roi_left=770
-death_roi_width=380
-death_roi_height=75
-corpse_x=950
-corpse_y=480
+death_roi_top=193
+death_roi_left=513
+death_roi_width=253
+death_roi_height=50
+corpse_x=633
+corpse_y=320
 ; waypoint
-wp_act_i_btn_x=135
-wp_act_i_btn_y=135
-wp_act_btn_width=100
-wp_first_btn_x=340
-wp_first_btn_y=193
-wp_btn_height=70
+wp_act_i_btn_x=90
+wp_act_i_btn_y=90
+wp_act_btn_width=67
+wp_first_btn_x=227
+wp_first_btn_y=129
+wp_btn_height=47
 ; pickit
-item_dist=400
-item_height_min=9
-item_width_min=30
+item_dist=267
+item_height_min=6
+item_width_min=20
 ; pather
-reached_node_dist=150
-min_walk_dist=38
-max_walk_dist=190
+reached_node_dist=100
+min_walk_dist=25
+max_walk_dist=127
 
-[ui_roi_1920_1080]
+[ui_roi]
 ; all rois are in  [left, top, width, height] format
-is_overburdened=17,765,470,90
-save_and_exit=750,350,410,260
-go_btn=800,940,320,100
-normal_btn=840,420,245,70
-nightmare_btn=840,490,245,70
-hell_btn=840,550,245,70
-gold_btn=1390,736,400,150
-health_globe=240,870,360,210
-mana_globe=1330,870,360,210
-cut_skill_bar=0,0,1920,980
-merc_icon=0,0,150,150
-loading_left_black=0,0,500,1080
-death=770,290,380,75
-tp_search=530,180,820,600
-repair_btn=477,709,135,120
-inventory=1290,515,591,247
-vendor_stash=44,126,600,600
-skill_right=996,1010,62,62
-hero_selection_logo=0,0,550,830
-play_btn=640,920,320,100
+is_overburdened=11,510,313,60
+save_and_exit=500,233,273,173
+go_btn=533,627,213,67
+normal_btn=560,280,163,47
+nightmare_btn=560,327,163,47
+hell_btn=560,367,163,47
+gold_btn=927,491,267,100
+health_globe=160,580,240,140
+mana_globe=887,580,240,140
+cut_skill_bar=0,0,1280,653
+merc_icon=0,0,100,100
+loading_left_black=0,0,333,720
+death=513,193,253,50
+tp_search=353,120,547,400
+repair_btn=318,473,90,80
+inventory=860,343,394,165
+vendor_stash=29,84,400,400
+skill_right=664,673,41,41
+hero_selection_logo=0,0,367,553
+play_btn=427,613,213,67
 
 [path]
-pindle_save_dist=1382,53, 1685,105, 1440,132
-eldritch_save_dist=978,95, 845,109
-pindle_end=1555,261
-eldritch_end=1012,42
-shenk_end=1600,816
+pindle_save_dist=921,35, 1123,70, 960,88
+eldritch_save_dist=652,63, 563,73
+pindle_end=1037,174
+eldritch_end=675,28
+shenk_end=1067,544
+eldritch_save_tp=640,533
+pindle_save_tp=400,27
+shenk_save_tp=437,215
```

Comment:
The save_tp positions are not used anymore, can be deleted

Reply:
done